beanprice/price.py

```diff
@@ -125,7 +133,7 @@ def parse_source_map(source_map_spec):
     Raises:
       ValueError: If an invalid pattern has been specified.
     """
-    source_map = collections.defaultdict(list)
+    source_map: Dict[str, List[PriceSource]] = collections.defaultdict(list)
```

Comment:
Why this? I don't think it is particularly useful.

Reply:
If this is not included, `mypy` fails with:

```
beanprice/price.py:136: error: Need type annotation for 'source_map'
```